src/main/features/response/guards/oweSomePaidNoneResponseRequiredGuard.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express'
+
+import { ResponseType } from 'response/form/models/responseType'
+import { Paths } from 'response/paths'
+import User from 'idam/user'
+
+export default class OweSomePaidNoneResponseRequiredGuard {
```

Comment:
Please do not use default import. Using default imports make refactoring / code maintenance slightly more difficult. Please change to named import.